database/seeders/TagTableSeeder.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class TagTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $tags = array();
+        for ($i = 1; $i < 4; $i++) {
+            $tag = array();
+            $tag['id'] = $i;
+            $tag['name'] = 'tag:' . $i;
+            $tag['created_at'] = date('Y-m-d H:i:s');
+            $tag['updated_at'] = date('Y-m-d H:i:s');
+            array_push($tags, $tag);
```

Comment:
```suggestion
            $tags[] = $tag;
```
語法可以做到的就不 call function